modules/nf-core/ampcombi/main.nf

```diff
@@ -0,0 +1,48 @@
+process AMPCOMBI {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::ampcombi=0.1.5" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ampcombi:0.1.5--pyhdfd78af_0':
+        'quay.io/biocontainers/ampcombi:0.1.5--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta),  path(amp_input)
+    path(faa_folder)
+
+    output:
+    tuple val(meta), path("*/*")                       , emit: sample_dir
+    tuple val(meta), path("*/*diamond_matches.txt")    , emit: txt
+    tuple val(meta), path("*/*ampcombi.csv")           , emit: csv
+    tuple val(meta), path("*/*amp.faa")                , emit: faa
+    path("AMPcombi_summary.csv")                       , optional:true, emit: summary_csv
+    path("AMPcombi_summary.html")                      , optional:true, emit: summary_html
+    path("*.log")                                      , emit: log
+    path("*/amp_ref_database")                         , optional:true, emit: results_db
+    path("*/amp_ref_database/*.dmnd")                  , optional:true, emit: results_db_dmnd
+    path("*/amp_ref_database/*.clean.fasta")           , optional:true, emit: results_db_fasta
+    path("*/amp_ref_database/*.tsv")                   , optional:true, emit: results_db_tsv
+    path "versions.yml"                                , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def fileoption = amp_input instanceof List ? "--path_list '${amp_input.collect{"$it"}.join("' '")}'" : "--amp_results $amp_input/"
```

Comment:
Is the `join` still meant to have the single quotes here, I though this was one of our bugs on Weds..?

Reply:
Well, we tried all options (single, double, none) and it made no difference. The bug was in the AMPcombi-code ...